src/api/Jalasoft.TeamUp.Resumes.API/Controllers/PutResumeSkill.cs

```diff
@@ -30,7 +30,7 @@ public PutResumeSkill(IResumesService resumesService)
         [OpenApiOperation(operationId: "UpdateResumeSkill", tags: new[] { "ResumeSkills" })]
         [OpenApiRequestBody("application/json", typeof(Skill[]), Description = "JSON request body containing list of skills")]
         [OpenApiParameter(name: "idResume", In = ParameterLocation.Path, Required = true, Type = typeof(int), Description = "The resume identifier.")]
-        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Skill[]), Description = "Successful response")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Resume), Description = "Successful response")]
```

Comment:
if the body sent for the put is a skills array, why an entire resumes object is returned?
if we want to return an entire resume object I think it would be better to have a put that receives the whole resume object